supporting_content/dimensions/resourceid.md

```diff
@@ -0,0 +1,55 @@
+| AWS       | Tax, API call charges etc.                        | (null)                                                                                           |
+| GCP       | Resource based                                    | //container.googleapis.com/projects/<project_id>/locations/<location>/clusters/<cluster>         |
+| GCP       | Non-resource based (e.g. network egress)          | (null)                                                                                           |
+| Microsoft | Resource group resources                          | /subscriptions/<guid>/resourceGroups/<name>/providers/<provider>/<resource-type>/<resource-name> |
+| Microsoft | Subscription resources                            | /subscriptions/<guid>/providers/<provider>/<resource-type>/<resource-name>                       |
+| Microsoft | Tenant resources                                  | /providers/<provider>/<resource-type>/<resource-name>                                            |
+| Microsoft | Purchase charges                                  | (null)                                                                                           |
+| Microsoft | Reservation purchases                             | /providers/Microsoft.Capacity/reservationOrders/<guid>                                           |
+| Microsoft | Savings plan purchases                            | /providers/Microsoft.Capacity/savingsPlanOrders/<guid>                                           |
+
+### Documentation
+
+AWS:
+> <https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html>
+
+Interesting note on the "The resource identifier" which is at the end of the ARN: This is the name of the resource, the ID of the resource, or a [resource path](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html#arns-paths). Some resource identifiers include a parent resource (sub-resource-type/parent-resource/sub-resource) or a qualifier such as a version (resource-type:resource-name:qualifier)."
+
+Microsoft:
+
+GCP:
+> <https://cloud.google.com/apis/design/resource_names> <https://cloud.google.com/asset-inventory/docs/resource-name-format> 
+
+## Discussion / Scratch space
+
+- Use one column for resource ID and make strong recommendations for the value to be globally unique within the provider. Eg. Full ARN for AWS.
+
+- Should this be prefixed with 'Vendor' e.g. VendorResourceID to show that it's a 1:1 mapping to a vendor provided column. 
+
+- Should governance say "Nullable where cost is not associated with a resource"
+
+- Do we make any statements about uniqueness? Or is this where we request a uniqueness composite key to be specified in a schema document
+
+- Can uniqueness and can it change over time?
+
+- Empty values should not be allowed if a user/systematically generated value is null (should this be an attribute specified as a higher level attribute OR within this scope)?
```

Comment:
Can we remove this given the null handling rules define empty values are not allowed?

Reply:
Same as above, i think the goal is to show the things we discussed (sausage making process). Happy to discuss and change if we feel this doc / discussions space should be more buttoned down.